content/en/cd-as-a-service/concepts/deployment/how-hpa-works.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "How CD-as-a-Service implements Kubernetes HPA"
+linktitle: "HPA"
+description: >
+  Armory CD-as-a-Service supports Kubernetes deployments with HPA.
+---
+ ## What is HPA?
+ Kubernetes Horizontal Pod Autoscaling (HPA) scales deployments in reaction to changes in observed metrics. When the deployment load increases, the number of deployed pods is increased to meet the demand. The scale behavior is horizontal, rather than vertical, which means that rather than assigning an existing pod more memory or CPU to handle the incoming traffic, new pods that identical instances of your application, are created.
```

Comment:
```suggestion
 Kubernetes Horizontal Pod Autoscaling (HPA) scales deployments in reaction to changes in observed metrics. When the deployment load increases, the number of deployed pods is increased to meet the demand. The scale behavior is horizontal, rather than vertical.  HPA scales a resource by adding or removing identical instances of your app instead of Kubernetes assigning an existing pod more resources to handle the incoming traffic.
```

Reply:
Please see re-write